package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slice-and-dice/register-a-food-business-validation",
-  "version": "1.15.0",
+  "version": "1.15.1",
```

Comment:
why not 1.16.0?

Reply:
This should indeed be 1.16.0, thanks for pointing out